webapp/packages/plugin-sql-editor/src/SqlEditor/useSqlEditor.ts

```diff
@@ -198,24 +202,22 @@ export function useSqlEditor(state: ISqlEditorTabState): ISQLEditorData {
       }
 
       const { connectionId, id, defaultSchema, defaultCatalog } = this.dataSource.executionContext;
+      const key = [connectionId, id, defaultSchema, defaultCatalog, position, word.slice(0, 1)];
+      const reset = this.hintsLimitIsMet || !simple;
```

Comment:
changes ctrl+space behaviour, please check that it works correctly (after/before *, new lines, multi-line queries)